conf/smRNA.config

```diff
@@ -1,143 +1,63 @@
 
 /*
  * -------------------------------------------------
- *  Nextflow config file for small RNA reference paths
+ * Nextflow config file for small RNA reference paths
  * -------------------------------------------------
  * Defines reference small RNA sequences genomes.
  */
 
 params {
   smrna {
+    'human' {
+      smrna_fasta = "$baseDir/assets/small_rna/Homo_sapiens/Homo_sapiens.smallRNA.fa.gz"
```

Comment:
I think we should move all of these files to nf-core/test-datasets instead of the pipeline repo. It means the pipeline itself is as lightweight as possible. You will just need to change the links here and this config can also be re-used by others if required 🎉

Reply:
Could we keep them with the pipeline if possible for now - they are only 192 Kb? Will also be published as part of an upcoming paper for the tRNAs.

Reply:
Sure 👍🏽